ProductsWebApi/ProductsWebApi/Service/ProductService.cs

```diff
@@ -0,0 +1,43 @@
+﻿using ProductsWebApi.Interface;
+using ProductsWebApi.Models;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace ProductsWebApi.Services
+{
+    public class ProductsService : IProductsService
+    {
+        private readonly IProductsService _productsService;
+
+        public ProductsService(IProductsService productsService)
```

Comment:
What's this???
Why are injecting IProductService in service. 🤔 
Shouldn't this be repository?